docs/docs/0.3.0/policies/README.md

```diff
@@ -206,6 +209,14 @@ spec:
         service: redis
         version: '2.0'
 ```
+You also need to add labels to your services and use that label as a matching tag. Above, you  see that we utilize the `version` tag to help with canary deployment. Another common use-case is to add a `env` tag as you separate testing, staging, and production environments' services.
```

Comment:
Let's only use term `tag` (and don't use `label`).

Reply:
> You also need to add labels to your services

What is "services" ? Is it a `Dataplane` resource ?

Reply:
Let's move this high-level description of how `TrafficRoute` works above (before the example).

Reply:
updated and moved up